File: nelio-alves/algoritmo-e-logica-de-programacao/python/exercicios/lanchonete.py
codigoProduto: int; quantidade: int
valorPagamento: float

codigoProduto = int(input("Codigo do produto comprado: "))
quantidade = int(input("Quantidade comprada: "))

valorPagamento = 0
if codigoProduto == 1:
	valorPagamento = quantidade * 5.00
elif codigoProduto == 2:
	valorPagamento = quantidade * 3.50
elif codigoProduto == 3:
	valorPagamento = quantidade * 4.80
elif codigoProduto == 4:
	valorPagamento = quantidade * 8.90
elif codigoProduto == 5:
	valorPagamento = quantidade * 7.32

print(f"Valor a pagar: R$ {valorPagamento:.2f}")

File: nelio-alves/algoritmo-e-logica-de-programacao/c/exercicios/lanchonete.c
#include <stdio.h>

int main() {
    int codigoProduto, quantidade;
    double valorPagamento;

    printf("Codigo do produto comprado: ");
    scanf("%d", &codigoProduto);

    printf("Quantidade comprada: ");
    scanf("%d", &quantidade);

    switch (codigoProduto) {
        case 1 :
            valorPagamento = quantidade * 5.00;
            break;

        case 2 :
            valorPagamento = quantidade * 3.50;
            break;

        case 3 :
            valorPagamento = quantidade * 4.80;
            break;

        case 4 :
            valorPagamento = quantidade * 8.90;
            break;

        case 5 :
            valorPagamento = quantidade * 7.32;
            break;
    }

    printf("Valor a pagar: R$ %.2lf\n", valorPagamento);

    return 0;
}
